dm/dm/config/subtask.go

```diff
@@ -411,11 +411,11 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 		// add suffix
 		var dirSuffix string
 		if isS3 {
-			dirSuffix = c.Name + "." + c.SourceID
+			dirSuffix = "/" + c.Name + "." + c.SourceID
```

Comment:
please add comment for this change

Reply:
i suddenly find that we shoud adapt this change when convert subtask to OpenAPITask
see here https://github.com/pingcap/tiflow/blob/50c98a6ac9eb456ac1cd47d871349b92976eb868/dm/dm/config/task_converters.go#L358

Reply:
i add trim path suffix at [this commit](https://github.com/pingcap/tiflow/pull/4897/commits/b481c0f6b746688b12d48798538bc9e49ea34c52), please take a look.